bastionzero/target/attributes.go

```diff
@@ -233,3 +234,46 @@ func FlattenControlChannelSummary(ctx context.Context, apiObject *targets.Contro
 		})
 	}
 }
+
+type DatabaseAuthenticationConfig struct {
```

Comment:
Let's rename `DatabaseAuthenticationConfig`, a TF model, to `DatabaseAuthenticationConfigModel` (`Model` suffix) to be consistent with the other TF model struct names in this project